README.md

```diff
@@ -17,17 +17,16 @@ track of certain things, especially as a project evolves over time, which is wha
 
```

Comment:
Also mention security advisories and unmaintained crates up here on the top level? Think it would be great to get a good overview

Reply:
Yes, definitely!